packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -33,7 +33,7 @@ module.exports = (resolve, rootDir, srcRoots) => {
     ],
     // where to search for files/tests
     roots: srcRoots.map(toRelRootDir),
-    testEnvironment: 'node',
+    testEnvironment: 'jsdom',
```

Comment:
This can actually be omitted, because that's the Jest's default, but decided to left as is for now.